runtime/src/iree/hal/drivers/vulkan/native_executable.cc

```diff
@@ -71,7 +71,7 @@ static iree_status_t iree_hal_vulkan_create_pipelines(
       executable_params->constant_count * sizeof(VkSpecializationMapEntry);
   size_t subgroup_control_size =
       pipeline_count *
-      sizeof(VkPipelineShaderStageRequiredSubgroupSizeCreateInfo);
+      sizeof(VkPipelineShaderStageRequiredSubgroupSizeCreateInfoEXT);
```

Comment:
This feels counter-intuitive to me. Is using the `EXT` symbols really best practice? Doesn't our version of Vulkan-Headers include the alias? Is some other definition of the headers getting used? (As a library, using the user/system-provided headers seems natural, but backwards-compatible aliases should be harmless to always include...)

https://github.com/openxla/iree/blob/5d15caf4bc2eefbc517461a2d78ae9b21008f40c/CMakeLists.txt#L844-L846 https://github.com/openxla/iree/blob/5d15caf4bc2eefbc517461a2d78ae9b21008f40c/runtime/src/iree/hal/drivers/vulkan/CMakeLists.txt#L84-L95 https://github.com/openxla/iree/blob/5d15caf4bc2eefbc517461a2d78ae9b21008f40c/runtime/src/iree/hal/drivers/vulkan/vulkan_headers.h#L44

Reply:
Cool. Yeah I see the issue now. I don't think there is a "best practice" regarding EXT / non-EXT symbols, esp. when they are strict aliases. Non-EXT symbols are widely available so I'd prefer to use it. Also we typically check the extension existence instead of requiring Vulkan 1.3 entirely. So using EXT symbols also matches that.